app/templates/home.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
```

Comment:
CSS doesn't work on any pages, i would check how some pages in master fixed this issue